.github/workflows/reusable_checks_rust.yml

```diff
@@ -109,8 +136,28 @@ jobs:
         with:
           pixi-version: v0.39.0
 
+      # Install the Vulkan SDK, so we can use the software rasterizer.
+      # TODO(andreas): It would be nice if `setup_software_rasterizer.py` could do that for us as well (note though that this action here is very fast when cached!)
+      - name: Install Vulkan SDK
+        if: ${{ matrix.name != 'macos' }}
+        uses: jakoch/install-vulkan-sdk-action@v1.0.5
+        with:
+          vulkan_version: ${{ env.VULKAN_SDK_VERSION }}
+          install_runtime: true
+          cache: true
+          stripdown: true
+
+      - name: Setup software rasterizer
+        run: pixi run python ./scripts/ci/setup_software_rasterizer.py
+
       - name: Download test assets
         run: pixi run python ./tests/assets/download_test_assets.py
 
-      - name: pixi run cargo test --all-targets --all-features
+      - name: Run tests (`cargo test --all-targets --all-features`)
+        if: ${{ inputs.CHANNEL != 'nightly' }}
+        # Need to use pixi due to NASM dependency.
         run: pixi run cargo test --all-targets --all-features
```

Comment:
Why run all the tests on all the three platforms for each PR?
We could save some money by only running them on Linux, and running Mac/Windows only on `main` and `nightly` builds.

Reply:
`pr` only runs `linux` ci, this is filtered in the line above here first
https://github.com/rerun-io/rerun/pull/8620/files#diff-f8fa90b87fca261d70633935468d767dfaefd6506376ab4d743c835404ad86d7R119
So exactly as you suggest :)

tbh I went with `if: ${{ inputs.CHANNEL != 'nightly' }}` because that made it easier to commen tout the filter above for testing purposes. I'll better repeat the exact same rule again to avoid confusion!